data/crac-creation/crac-creator-cse/src/main/java/com/powsybl/openrao/data/craccreation/creator/cse/criticalbranch/CriticalBranchReader.java

```diff
@@ -134,12 +140,37 @@ private void importPreventiveCnec(TBranch tBranch, UcteFlowElementHelper branchH
         importCnec(crac, tBranch, branchHelper, isMonitored ? tBranch.getIlimitMNE() : tBranch.getImax(), null, crac.getPreventiveInstant().getId(), isMonitored);
     }
 
-    private void importCurativeCnecs(TBranch tBranch, UcteFlowElementHelper branchHelper, String outage, Crac crac, boolean isMonitored) {
+    /**
+     * Returns true if at least one CNEC has been correctly imported.
+     * @param tBranch
+     * @param branchHelper
+     * @param outage
+     * @param crac
+     * @param isMonitored
+     * @return
+     */
+    private boolean importCurativeCnecs(TBranch tBranch, UcteFlowElementHelper branchHelper, String outage, Crac crac, boolean isMonitored) {
         HashMap<Instant, TImax> cnecCaracs = new HashMap<>();
         cnecCaracs.put(crac.getOutageInstant(), isMonitored ? tBranch.getIlimitMNEAfterOutage() : tBranch.getImaxAfterOutage());
-        cnecCaracs.put(crac.getInstant(InstantKind.AUTO), isMonitored ? tBranch.getIlimitMNEAfterSPS() : tBranch.getImaxAfterSPS());
-        cnecCaracs.put(crac.getInstant(InstantKind.CURATIVE), isMonitored ? tBranch.getIlimitMNEAfterCRA() : tBranch.getImaxAfterCRA());
+        cnecCaracs.put(crac.getInstant(InstantKind.AUTO), isMonitored ? tBranch.getIlimitMNEAfterSPS() : findTargetImaxAfterSPS(tBranch));
+        cnecCaracs.put(crac.getInstant(InstantKind.CURATIVE), isMonitored ? tBranch.getIlimitMNEAfterCRA() : findTargetImaxAfterCRA(tBranch.getImaxAfterCRA()));
         cnecCaracs.forEach((instant, iMax) -> importCnec(crac, tBranch, branchHelper, iMax, outage, instant.getId(), isMonitored));
+        return cnecCaracs.values().stream().anyMatch(Objects::nonNull);
+    }
+
+    private TImax findTargetImaxAfterCRA(final TImax imaxAfterCRA) {
+        return imaxAfterCRA != null ? imaxAfterCRA : createDefaultImaxAfterCRA();
+    }
+
+    private TImax createDefaultImaxAfterCRA() {
+        final TImax tempImax = new TImax();
+        tempImax.setUnit("Pct");
+        tempImax.setV((short) 100);
+        return tempImax;
+    }
+
+    private TImax findTargetImaxAfterSPS(final TBranch tBranch) {
+        return tBranch.getImaxAfterSPS() != null ? tBranch.getImaxAfterSPS() : tBranch.getImaxAfterOutage();
```

Comment:
I understand that this is required by the business requirements. However, I'm not convinced that it is a good idea, since the CSE CC process does not have any SPS. It would only duplicate the CNECs in outage instant to auto instant, with no effect on the RAO.
What do you think @phiedw ?